packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -212,6 +212,62 @@ export async function fetchCertificate(
   }
 }
 
+/**
+ * Fetch a single destination by name without credentials.
+ * @internal
+ */
+export async function fetchDestinationByNameWithoutTokens(
+  destinationServiceUri: string,
+  serviceToken: string,
+  options: DestinationServiceOptions
+): Promise<Omit<Destination, 'AuthAndExchangeTokens'> | undefined> {
+  const targetUri = `${removeTrailingSlashes(
+    destinationServiceUri
+  )}/destination-configuration/v1/destinations/${
+    options.destinationName
+  }?$skipCredentials=true`;
+
+  if (options?.useCache) {
+    const destinationsFromCache =
+      destinationServiceCache.retrieveDestinationsFromCache(
```

Comment:
[q] So we're caching for `$?skipCredentials=true`. Not sure I understand why?

Reply:
I don't understand the use-case well enough to answer this question. The underlaying assumption is that in the second call of this destination, the credentials are required?

Reply:
In the old implementation the `fetchDestinations` also had a cache with a fixed life time of 300.000 ms because the it only returned the fixed data for the destination where the destination service has also an internal cache of 5 minutes. The `skipCredentials=true` is somehow similar as it also returns just the header data but for a single destination instead of the list.